kubernetes/doc/create-options.md

```diff
@@ -0,0 +1,122 @@
+# Kubernetes createOptions Extensions
+
+IoT Edge on Kubernetes supports several experimental features to allow a user to assign additional parameters to pods that represents IoT Edge Modules.
```

Comment:
>represents [](start = 123, length = 10)

Consider rephrasing this to clarify the pods represent modules, not the parameters.